src/dbus/server/introspect.xml

```diff
@@ -246,8 +246,15 @@
       <arg name="epskc" type="s" direction="out"/>
     </method>
 
-    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode. -->
+    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode.
+      @retain_active_session: 
+        "false" - Default value. Disconnects the Border Agent from any active secure sessions.
+                  Ephemeral key would be cleared after the session is disconnected.
+        "true" - Clears ephemeral key only when Border Agent state is running
+                 and no active session established.
```

Comment:
The description is not accurate. The ephemeral key is always cleared no matter there is active session or not.

Reply:
Modified accordingly. Thanks!